app/src/pages/inside/projectSettingsPageContainer/utils.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2023 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import DOMPurify from 'dompurify';
+
+export const createLink = (data, href) =>
```

Comment:
Please rename it to the `createExternalLink` as it leads to the external resources.
Also I suggest to move this utility upper (I guess in the future we will provide external links not only in Project settings) to the common folder and cover with tests.